### PR TITLE
BAQE-1997

### PR DESCRIPTION
**Pull request by dupliaka**
Bumped version to 8.11.0-SNAPSHOT
Updated paths in Solver config
Added jboss repository to test SNAPSHOTs

### CODE REVIEW

File: optaplanner-8-benchmarks/jmh-result.csv [hunk 1]
```diff
@@ -0,0 +1,163 @@
+"Benchmark","Mode","Threads","Samples","Score","Score Error (99.9%)","Unit","Param: constructionHeuristicType","Param: dataSet","Param: dataset","Param: moveThreadCount"
```
Comment (rsynek) [inline optaplanner-8-benchmarks/jmh-result.csv:1]: Is adding this JMH result file intentional?

**Review by rsynek — APPROVED** [review]
LGTM, just the JMH file looks suspicious.

File: optaplanner-8-benchmarks/jmh-result.csv (hunk 1 above)
Comment (dupliaka) [reply optaplanner-8-benchmarks/jmh-result.csv]: My fault, fixed

**Comment by dupliaka** [comment]
Java 8 check is falling since OptaPlanner is not supporting compilation on java 1.8 anymore. 
PR check change would be a matter of another PR

**Pull request merged**